cadc-log/src/main/java/ca/nrc/cadc/log/WebServiceLogInfo.java

```diff
@@ -279,4 +310,18 @@ public void setRunID(String runID) {
         this.runID = runID;
     }
 
+    public void setClass(Class clz) {
+        this.clz = clz;
```

Comment:
Could the calling class not be determined from the call stack in start() and end()?

Reply:
Removed the class from the logging, duplicate information.